backend/src/common/client/api_server/run_client_fake.go

```diff
@@ -74,3 +74,14 @@ func (c *RunClientFake) ListAll(params *runparams.ListRunsParams, maxResultSize
 	[]*runmodel.APIRun, error) {
 	return listAllForRun(c, params, maxResultSize)
 }
+
+func (c *RunClientFake) Terminate(params *runparams.TerminateRunParams) error {
```

Comment:
We'll need an integration test for this, can you add a TODO and open an issue to track it?